private/bufpkg/bufimage/bufimage.go

```diff
@@ -340,49 +343,6 @@ func CloneImageFile(imageFile ImageFile) (ImageFile, error) {
 	)
 }
 
-// MergeImages returns a new Image for the given Images. ImageFiles
```

Comment:
The merge functionality was completely unused (I also checked in our internal repo with BSR code); it is only ever invoked from test code.

So I removed it. (I came across it because it has a call to `newImage` that would need to be updated.)